org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/WorkspaceManager.xtend

```diff
@@ -104,8 +107,16 @@ import org.eclipse.xtext.ide.server.BuildManager.Buildable
         val result = buildManager.doInitialBuild(newProjects, cancelIndicator)
         afterBuild(result)
     }
+    
+    protected def IWorkspaceConfig getWorkspaceConfig() {
+    	if (workspaceConfig === null) {
```

Comment:
Should we rename the field to e.g. `_workspaceConfig`, to make sure it is always accessed through the getter?